File: scratch/check_words.js
const fs = require('fs');
const content = fs.readFileSync('lib/blog-posts.ts', 'utf8');

const posts = content.split('slug: "').slice(1);
posts.forEach(post => {
    const slug = post.split('"')[0];
    const match = post.match(/contentHtml:\s*([^]*)/);
    if (match) {
        const html = match[1];
        const text = html.replace(/<[^>]+>/g, ' ');
        const words = text.split(/\s+/).filter(w => w.length > 0).length;
        console.log(slug + ': ' + words + ' words');
    }
});

File: scratch/check_words.py
import re

with open('lib/blog-posts.ts', 'r', encoding='utf-8') as f:
    content = f.read()

posts = content.split('slug: "')[1:]
for post in posts:
    slug = post.split('"')[0]
    html_match = re.search(r'contentHtml:\s*([^]*)', post)
    if html_match:
        html = html_match.group(1)
        text = re.sub(r'<[^>]+>', '', html)
        words = len(text.split())
        print(f'{slug}: {words} words')
